### PR TITLE
IL-EPIC6-US6 AI integration

### PR DESCRIPTION
**Pull request by nnick44**

### CODE REVIEW

File: src/main/java/com/zufar/icedlatte/ai/api/AiChecker.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+            log.debug("Response Code: " + responseCode);
+
+            BufferedReader in = new BufferedReader(new InputStreamReader(con.getInputStream()));
+            String inputLine;
+            StringBuilder response = new StringBuilder();
+            while ((inputLine = in.readLine()) != null) {
+                response.append(inputLine);
+            }
+            in.close();
```
Comment (annstriganova) [inline src/main/java/com/zufar/icedlatte/ai/api/AiChecker.java:48]: I believe it should be in `finally` section, consider using `try-with-resources`

**Review by annstriganova — CHANGES_REQUESTED** [review]
Reviewed

File: src/main/java/com/zufar/icedlatte/ai/api/AiChecker.java (hunk 1 above)
Comment (nnick44) [reply src/main/java/com/zufar/icedlatte/ai/api/AiChecker.java]: yes, fix it

**Review by annstriganova — APPROVED** [review]
Approved.

**Review by Sunagatov — APPROVED** [review]

**Pull request merged**